features/networking/pod.feature

```diff
@@ -190,3 +190,153 @@ Feature: Pod related networking scenarios
+      | hostname -i |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_ip clipboard
+    
+    Given I select a random node's host
+    Given I have a project
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    When I run the :create client command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/egress-http-proxy/egress-http-proxy-pod.yaml |
+    Then the pod named "egress-http-proxy" becomes ready
+
+    #Pod should not access MCS via an egress router pod
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22623/config/master | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22624/config/master | -k |
+    Then the output should contain "Connection refused"
+
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23893
+  @admin
+  Scenario: A pod in a namespace with an egress IP cannot access the MCS
+    Given I use the first master host
+    #Step to obtain master IP
+    And I run commands on the host:
+      | hostname -i |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_ip clipboard
+    
+    Given I select a random node's host
+    And evaluation of `node.name` is stored in the :egress_node clipboard
+    #add the egress ip to the hostsubnet
+    And the valid egress IP is added to the "<%= cb.egress_node %>" node
+    Given I have a project
+    And evaluation of `project.name` is stored in the clipboard
+    # add the egress ip to the project
+    When I run the :patch admin command with:
+    | resource      | netnamespace                         |
+    | resource_name | <%= project.name %>                    |
+    | p             | {"egressIPs":["<%= cb.valid_ip %>"]} |
+    | type          | merge                                |
+    Then the step should succeed
+    #pod-for-ping will be a non-hostnetwork pod
+    And I have a pod-for-ping in the project
+    
+    #Pod cannot access MCS
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22623/config/master | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22624/config/master | -k |
+    Then the output should contain "Connection refused"
+
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23894
+  @admin
+  Scenario: User cannot access the MCS by creating a service that maps to non-MCS port to port 22623 or 22624 on the IP of a master (via manually-created ep's)
+    Given I use the first master host
+    #Step to obtain master IP
+    And I run commands on the host:
+      | hostname -i |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_ip clipboard
```

Comment:
instead of lines 319 to 322, you can remove them, and where IP is needed, you can use `@host.local_ip`.

Reply:
So better use `env.master_hosts.first.local_ip`.

Reply:
Didn't aware of "env.master_hosts.first.local_ip". I was thinking to create a def for this at some point. Thans will give it a try

Reply:
Thanks @akostadinov it works https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3-smoke/391/console. Good to know about this short step. Guess i will go ahead and replace in all scenarios under this PR.